modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -0,0 +1,132 @@
+[id="proc-configuring-postgresql-instance-using-helm_{context}"]
+= Configuring an external PostgreSQL instance using the Helm Chart
+
+You can configure an external PostgreSQL instance by using the Helm Chart. By default, the Helm Chart creates and manages a local instance of PostgreSQL in the same namespace where you have deployed the {product-very-short} instance. However, you can change this default setting to configure an external PostgreSQL database server, for example, AWS RDS or Azure database. 
+
+.Prerequisites
+
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+
+* You have a PostgreSQL certificate of the .pem format so that you can secure your database connection using the Secure Socket Layer (SSL) or Transport Layer Security (TLS) protocol. For more information, see link:{LinkAdminGuide}#postgresql-certificate-for-aws-rds_admin-rhdh[Downloading PostgreSQL certificate for AWS RDS] or link:{LinkAdminGuide}#generating-postgreSQL-certificate-for-azure-database_admin-rhdh[Generating PostgreSQL certificate for Azure database].
+
+[NOTE]
+====
+By default, {product-short} uses a database for each plugin and automatically creates it if none is found. As an administrator, know that users might need the `Create Database` privilege in addition to `PSQL Database` privileges for configuring an external PostgreSQL instance.  
+====
+ 
+
+.Procedure
+
+. Create a credential secret to connect with the PostgreSQL instance:
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: <cred-secret> <1>
+type: Opaque
+stringData: <2>
+ POSTGRES_PASSWORD: <password>
+ POSTGRES_PORT: "<db-port>"
+ POSTGRES_USER: <username>
+ POSTGRES_HOST: <db-host>
+ PGSSLMODE: <ssl-mode> # for TLS connection <3>
+ NODE_EXTRA_CA_CERTS: <abs-path-to-pem-file> # for TLS connection, e.g. /opt/app-root/src/postgres-crt.pem <4>
+EOF
+----
+<1> Provide the name of the credential secret.
+<2> Provide credential data to connect with your PostgreSQL instance.
+<3> Optional: Provide the value only if you need a TLS connection for your PostgreSQL instance.
```

Comment:
`Provide the value` sounds a bit vague to me. 
I suggest to do one of the following:
a. point to postgres documentation for SSL modes: https://www.postgresql.org/docs/15/libpq-connect.html#LIBPQ-CONNECT-SSLMODE
b. say that they have to provide the value according to desired SSL mode

This applies to all the occurrences of `PGSSLMODE`